.datignore

```diff
@@ -1,2 +1,5 @@
 node_modules
 dat.json
+*~
+*.swp
+.datignore
```

Comment:
I put those in `.datignore` on purpose, since the git checkout and the dat archive are the same directory on my computer.
Since BeakerBrowser does not (yet) support updating a selection of modified files, it would mean that I would commit the temporary files to the dat archive in addition to the files I modified.

Let me know, if there is a better way of doing this.